src/bci_build/templates.py

```diff
@@ -13,6 +13,7 @@
 #!BuildTag: {{ tag }}
 {% endfor -%}
 {% if image.build_version %}#!BuildVersion: {{ image.build_version }}{% endif %}
+{% if image.build_release %}#!BuildRelease: {{ image.build_release }}{% endif %}
```

Comment:
Can you please modify the template so that we don't get a mass change in the Dockerfiles?

Reply:
Done I think.